src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -19,10 +20,13 @@ public static partial class ViewExtensions
 
 		public static void UpdateIsEnabled(this UIView platformView, IView view)
 		{
-			if (platformView is not UIControl uiControl)
-				return;
+			if (platformView is UIControl uiControl)
+				uiControl.Enabled = view.GetIsEnabled();
+
+			if (platformView is LayoutView layout)
```

Comment:
We don't need to enable/disable the layout backing controls; the child controls will all be disabled anyway. We can just drop lines 26 & 27,